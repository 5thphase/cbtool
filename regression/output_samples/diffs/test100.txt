client_specweb, driver_filebench, driver_coremark, lostback, seed, coremark, driver_hadoop, cassandra, rubis_driver, predictablevm, ddgen, lb, ycsb, willfail, client_lost, client_daytrader, hadoopmaster, storageclient, driver_tradelite, specwebfront, was, netclient, storageserver, filebench, specjbb, client_windows, rubis_lb, driver_fbnfsrd, faildb2, fen_hpc, tinyvm, db2, rubis_appsrv, netserver, rubis_database, windows, driver_netperf, client_tradelite, driver_daytrader, lostfront, cn_hpc, specwebback, hadoopslave
