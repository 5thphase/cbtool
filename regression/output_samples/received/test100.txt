VMs with the following roles can be attached to to this experiment (Cloud TESTCLOUD) :
driver_filebench, driver_coremark, seed, coremark, mysql, cassandra, client_windows, ddgen, lb, iperfserver, ycsb, xpingsender, client_daytrader, redis, driver_daytrader, xpingreceiver, nuttcpclient, hadoopmaster, fioserver, giraphslave, driver_tradelite, was, netclient, nuttcpserver, filebench, specjbb, predictablevm, mongodb, fioclient, driver_fio, cn_hpc, fen_hpc, mongo_cfg_server, iperfclient, db2, giraphpmaster, netserver, faildb2, windows, geronimo, driver_netperf, client_tradelite, driver_hadoop, willfail, tinyvm, mongos, hadoopslave
