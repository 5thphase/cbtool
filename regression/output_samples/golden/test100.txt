VMs with the following roles can be attached to to this experiment (Cloud TESTCLOUD) :
driver_filebench, driver_coremark, predictablevm, seed, coremark, mysql, cassandra, client_windows, ddgen, lb, hadoopmaster, willfail, client_daytrader, driver_daytrader, fioserver, driver_fio, was, netclient, filebench, specjbb, driver_tradelite, mongodb, fioclient, cn_hpc, fen_hpc, mongo_cfg_server, db2, netserver, faildb2, windows, geronimo, ycsb, driver_netperf, client_tradelite, driver_hadoop, tinyvm, mongos, hadoopslave
